### PR TITLE
Upgrade free marker to 2.3.25 (from 2.3.22)

### PR DESCRIPTION
**Pull request by aledsage**
Freemarker 2.3.25 adds support for iterating over map entries, which will be useful for some entity implementations.

@ygy could you review this please?

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -137,7 +137,7 @@
         <jetty-schemas.version>3.1.M0</jetty-schemas.version>
         <airline.version>0.6</airline.version>
         <mockwebserver.version>20121111</mockwebserver.version>
-        <freemarker.version>2.3.22</freemarker.version>
+        <freemarker.version>2.3.25-incubating</freemarker.version>
```
Comment (bostko) [inline pom.xml]: I'd rather not upgrade to incubating release.

**Comment by bostko** [comment]
There are numerous ways to iterate a map. 
@ygy what is the use case which new freemarker supports?

File: pom.xml (hunk 1 above)
Comment (aledsage) [inline pom.xml]: @bostko the freemarker project was recently moved to Apache - all such projects start as "incubating", and must therefore have "-incubating" in their release names, until they graduate as a top-level project. It was the same for Apache Brooklyn.

The "-incubating" says nothing about the quality or maturity of the product/release.

**Comment by aledsage** [comment]
@bostko see http://freemarker.org/docs/versions_2_3_25.html, in the section "changes on the FTL side":

```
Extended the list directive to support listing hashes (such as Map-s), like <#list map as k, v>${k}: ${v}</#list>, where k and v are the key and value in the key-value pairs of the hash.
```

Also see http://freemarker.org/docs/versions_2_3_24.html, but I don't think those changes relate to iterating over maps.

**Comment by neykov** [comment]
LGTM, merging.

**Pull request merged**